src/Compilers/CSharp/Test/Semantic/Semantics/TargetTypedDefaultTests.cs

```diff
@@ -787,7 +859,7 @@ static void Main()
             var tree = comp.SyntaxTrees.First();
             var model = comp.GetSemanticModel(tree);
             var def = tree.GetCompilationUnitRoot().DescendantNodes().OfType<DefaultLiteralSyntax>().Single();
-            Assert.Null(model.GetTypeInfo(def).Type);
+            Assert.Equal("System.Int32?", model.GetTypeInfo(def).Type.ToTestDisplayString());
```

Comment:
>Assert.Equal("System.Int32?", model.GetTypeInfo(def).Type.ToTestDisplayString()); [](start = 12, length = 81)

It would be good to test conversion and constant value as well.